docs/source/quickstart.rst

```diff
@@ -38,49 +39,70 @@ Dependencies
 
 * `Enable <https://github.com/enthought/enable/>`_, a framework for writing
   interactive visual components, and for abstracting away GUI-toolkit-specific
-  details of mouse and keyboard handling
+  details of mouse and keyboard handling. This currently still depends on the
+  Python Image Library (PIL).
```

Comment:
I suspect the comment about PIL may be out of date (the Enable readme recommends using Pillow, at least). I would remove this.